README.md

```diff
@@ -31,6 +31,13 @@ The diagram below shows the message exchange between the involved parties.
 
 ![alt Kong OIDC flow](docs/kong_oidc_flow.png)
 
+For security purposes the following headers are stripped at the beginning of this plugins execution:
+* `X-Access-Token`
+* `X-ID-Token`
+* `X-Userinfo`
```

Comment:
are these hard coded, or are they part of some configurable list?  shouldn't the be configurable?

Reply:
@brainthinks these are definitely hardcoded - I don't think we should change them or make them configurable.

Don't think we should consider making them configurable till we got a use case.